src/utils/date.ts

```diff
@@ -10,3 +10,15 @@ export function fromNow(date: string) {
 export function getCurrentYear() {
   return new Date().getFullYear();
 }
+
+export function getFormatedDate(date: string, format?: string) {
+  const currentDate = dayjs(date);
+  if (!format) {
+    return [currentDate.date(), currentDate.month().toString(), currentDate.year().toString()].join('/');
+  }
+  const formatValues = format.toUpperCase().split('/');
+  const mappedDates = formatValues.map((event) =>
+    event === 'DD' ? currentDate.date() : event === 'MM' ? currentDate.month() : currentDate.year(),
+  );
+  return mappedDates.join('/');
+}
```

Comment:
Is there any specific reason we need to format it like this?

If I understood correctly, the API just returns dates as string and we can apply the formatting like following.
And call the method like `format("DD/MM")`

```suggestion
export function format(date: string, format = "DD/MM/YYYY") {
  return dayjs(date).format(format);
}
```

Reply:
Nope, I did not see how it worked. However, I'll keep this in mind.